Gemfile

```diff
@@ -3,4 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in mini_mime.gemspec
 gemspec
 
-gem "mime-types", "~> 3" if RUBY_VERSION > '2'
+gem 'mime-types', '~> 3' if RUBY_VERSION > '2'
+gem 'memory_profiler' if RUBY_VERSION >= '2.1.0'
+gem 'benchmark-ips'
+if RUBY_PLATFORM != 'java' && RUBY_VERSION > '1.9'
+  gem 'micro_mime', :git => 'https://github.com/stereobooster/micro_mime.git'
```

Comment:
The git source `git://github.com/stereobooster/micro_mime.git` uses the `git` protocol, which transmits data without encryption. Disable this warning with `bundle config git.allow_insecure true`, or switch to the `https` protocol to keep your data secure.